packages/garbo/src/fights.ts

```diff
@@ -1383,6 +1391,51 @@ const freeRunFightSources = [
     },
     freeRunConstraints(true),
   ),
+  // Use free runs and free kills to get our bofa wishes
+  new FreeRunFight(
+    () =>
+      have($skill`Just the Facts`) &&
+      get("_bookOfFactsWishes") < 3 &&
+      shouldDoBofaFishing() &&
+      findFreeKill() !== null &&
+      getBestBofaWishLocation() !== null &&
+      unlock(
+        getBestBofaWishLocation(),
+        (3 - get("_bookOfFactsWishes")) * garboValue($item`pocket wish`),
+      ),
+    (runSource: ActionSource) => {
+      const best = getBestBofaWishLocation();
+      if (!best) throw `Bofa wish location should exist, but doesn't`;
```

Comment:
```suggestion
      if (!best) throw new Error("Bofa wish location should exist, but doesn't");
```
We generally prefer Errors to strings when throwing, and we should only use backticks when we're doing template literal stuff